config/settings.defaults.coffee

```diff
@@ -256,7 +256,7 @@ module.exports =
 	
 	# Should we allow access to any page without logging in? This includes
 	# public projects, /learn, /templates, about pages, etc.
-	allowPublicAccess: false
+	allowPublicAccess: if process.env["SHARELATEX_ALLOW_PUBLIC_ACCESS"]? then true else false
```

Comment:
I think settings SHARELATEX_ALLOW_PUBLIC_ACCESS to false in env would result it in being true here?